CustomPlugin.Tests/ServiceClientFactoryTest.cs

```diff
@@ -0,0 +1,85 @@
+using Crud;
+using Environment;
+using Microsoft.AspNetCore.Http;
+using NFluent;
+using NUnit.Framework;
+using Service;
+
+namespace CustomPlugin.Tests
+{
+    public class ServiceClientFactoryTest
+    {
+        private ServiceClientFactory _serviceClientFactory;
+        private MiaEnvConfiguration _miaEnvConfiguration;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _miaEnvConfiguration = new MiaEnvConfiguration
+            {
+                USERID_HEADER_KEY = "1",
```

Comment:
can we use plausibile keys? such as `useridkey`, etc.. ?

same below at line 58

Reply:
Done